ZipExtractor-Bukkit/src/main/java/com/dscalzi/zipextractor/bukkit/ZipExtractor.java

```diff
@@ -45,7 +39,6 @@ public void onEnable() {
         ZServicer.initalize(ConfigManager.getInstance().getMaxQueueSize(),
                 ConfigManager.getInstance().getMaxPoolSize());
         this.getCommand("zipextractor").setExecutor(new MainExecutor(this));
-        metrics = new Metrics(this);
```

Comment:
Don't remove this.

Reply:
Why? `metrics` variable was not used at all, you even suppressed the warning of it not being used in the comment above.

Reply:
It's to turn on metrics with bStats. Just how their API works.

Reply:
I think the variable is not needed at all, i think doing `new Metrics(this)` would be enough.

Reply:
I'd rather have the variable there

Reply:
Re-added.